TensorFlowSharp/OperationsExtras.cs

```diff
@@ -460,7 +460,76 @@ public TFOutput ClipByAverageNorm (TFOutput x, TFOutput clip_norm, string operNa
 		}
 
 
+		/// <summary>
+		///   Computes sigmoid cross entropy given `logits`.
+		/// </summary>
+		/// 
+		/// <remarks>
+		///    Measures the probability error in discrete classification tasks in which each
+		///    class is independent and not mutually exclusive.For instance, one could
+		///    perform multilabel classification where a picture can contain both an elephant
+		///    and a dog at the same time.
+		/// </remarks>
+		/// 
+		public TFOutput sigmoid_cross_entropy_with_logits (TFOutput labels, TFOutput logits, string operName = null)
```

Comment:
This does not look right, the code below references a different method, would you mind renaming this to follow the .NET conventions for method names?